src/test/java/com/kryptnostic/datastore/edm/DatastoreTests.java

```diff
@@ -307,7 +328,7 @@ public void testAddPropertyToSchema(){
         	caught = t;
         }
         assertNotNull(caught);
-        assertSame(BadRequestException.class, caught.getClass());
+        assertSame(IllegalArgumentException.class, caught.getClass());
```

Comment:
This seems odd since you are asserting an exception must be thrown and it must be of type IllegalArgumentException. I would use either the test wide annotation or this check, but not both.